libecole/include/ecole/reward/dualbound.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include "ecole/reward/abstract.hpp"
+#include "ecole/scip/type.hpp"
+
+#define infinity   1e+20
+
+namespace ecole::reward {
+
+class DualBound : public RewardFunction {
+public:
+	void before_reset(scip::Model& model) override;
+	Reward extract(scip::Model& model, bool done = false) override;
+
+private:
+//	scip::real dual_bound_value = 0.0;
+	scip::real dual_bound_value = - infinity;
```

Comment:
This reward function does not need to keep a memory of the past, so I think this field is not necessary.